_layouts/default2025.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>{{ page.title }}</title>
+    <meta name="description" content="{{ page.description }}">
+    {% if page.stylesheet %}
+    <link rel="stylesheet" href="/assets/css/{{ page.stylesheet }}.css">
+    {% else %}
+    <link rel="stylesheet" href="/assets/css/styles.css">
+    {% endif %}
+    <link rel="icon" href="/assets/img/favicon.png" sizes="192x192">
+    <link rel="icon" href="/assets/img/favicon.ico" sizes="48x48">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
```

Comment:
You are missing quite a lot of functionality here:
https://github.com/openSUSE/jekyll-theme/blob/master/_includes/head.html

Reply:
Hmm, yes. I think I've reintroduced everything other than obsolete stylesheets and JavaScript.